model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -49,6 +53,10 @@ class PerfAnalyzer:
     # The metrics that PerfAnalyzer can collect
     perf_metrics = {
         PerfLatency: "_parse_perf_latency",
```

Comment:
This should no longer be called from here, so can be removed from this dispatch table

Reply:
Updated.